Textual improvements (#222)
openproblems-bio/website: documentation/contribute/add_a_dataset.qmd

```
:::

> For more parameter options, refer to @sec-parameters

Reviewer: Incomplete sentence

> For more parameter options, refer to @sec-parameters.

Reviewer 2: this causes the website to show `section ...` this has no meaning to the reader. I would suggest to just add a link.
```suggestion
For more parameter options, refer to [Parameters](#parameters) section.
```

## Step 3: Create a script

Next, create a script that will generate or load the dataset.
Here we show an example script that generates a random dataset, but check out `src/datasets/loaders` for real data examples.

> The script must ensure that that the output anndata object has the format as described in @sec-format.

Reviewer 2: same applies for this
```suggestion
The script must ensure that that the output anndata object has the format as described in the [format](#format-of-a-raw-dataset-object) section.

```

<!-- TODO: write a generic dataset loader unit test and use that instead, see #103 -->

> ## Parameters {#sec-parameters}

Reviewer 2: Not needed if changes are accepted.
```suggestion
## Parameters
```

:::

> ## Format of a raw dataset object {#sec-format}

Reviewer 2: same

```suggestion
## Format of a raw dataset object
```
